### PR TITLE
TEST: QA-10976 Add missing tests for Sorting

### PR DESCRIPTION
**Pull request by KhoaPham1010**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7612/ : FAILURE in 1m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3729/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7613/ : SUCCESS in 2m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3730/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7614/ : FAILURE in 1m 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3731/ : SUCCESS in 3s

**Comment by KhoaPham1010** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7615/ : FAILURE in 1m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3732/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7626/ : SUCCESS in 5m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3744/ : SUCCESS in 3s

**Comment by KhoaPham1010** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1635.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3745/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3357/ : FAILURE in 20m 11s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7627/ : FAILURE in 1m 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3746/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7628/ : SUCCESS in 3m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3747/ : SUCCESS in 3s

**Comment by KhoaPham1010** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1635.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3748/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3358/ : SUCCESS in 21m 10s

File: examples/src/components/DynamicSortingExample.jsx [hunk 1]
```diff
@@ -123,6 +123,21 @@ export class DynamicSortingExample extends Component {
                     }),
                 ],
             },
+            {
+                key: "serveral-element",
+                label: "Combine serveral sorting",
```
Comment (xMort) [inline examples/src/components/DynamicSortingExample.jsx:128]: Serveral? Is this typo of "several"?

File: examples/src/routes/PivotTableDemo.jsx [hunk 2]
```diff
@@ -27,41 +27,59 @@ export const PivotTableDemo = () => (
 
         <hr className="separator" />
 
-        <h2 id="measures-row-attributes-and-column-attributes">Example of Presorted Pivot Table</h2>
-        <ExampleWithSource for={() => <PivotTableSortingExample />} source={PivotTableSortingExampleSRC} />
-
+        <h2>Example of Presorted Pivot Table</h2>
```
Comment (xMort) [inline examples/src/routes/PivotTableDemo.jsx:30]: I would upper case just the first letter. This title and lots of after it seems inconsistent with other ones you have here.

File: examples/test/Sorting_test.js [hunk 3]
```diff
@@ -75,6 +76,14 @@ test("should display sorted graphs", async t => {
         .expect(dynamicSortingChartLegendLabels.textContent)
         .eql("TexasNew YorkFloridaAlabamaCalifornia")
 
+        // dynamic serveral element sorting
```
Comment (xMort) [inline examples/test/Sorting_test.js:79]: Maybe typo here as well?

File: examples/test/VisualizationByUri_test.js [hunk 4]
```diff
@@ -43,8 +43,33 @@ test("Table visualization should render", async t => {
         .eql("Month/Year (Date)");
 });
 
-test("Bar chart should render", async t => {
+test("Bar chart should render with data is sorted desc", async t => {
```
Comment (xMort) [inline examples/test/VisualizationByUri_test.js:46]: "Bar chart should render with data sorted in descending order"

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by xMort** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3997/ : UNSTABLE in 11m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/66/ : SUCCESS in 19s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7634/ : SUCCESS in 6m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1841/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2729/ : FAILURE in 19m 02s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3998/ : SUCCESS in 17m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/67/ : SUCCESS in 21s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7636/ : SUCCESS in 3m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1842/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2730/ : FAILURE in 20m 08s

**Comment by KhoaPham1010** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1635.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3766/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3364/ : FAILURE in 20m 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7658/ : SUCCESS in 6m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3776/ : SUCCESS in 4s

**Comment by KhoaPham1010** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1635.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3779/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3366/ : SUCCESS in 20m 03s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4009/ : SUCCESS in 13m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/77/ : SUCCESS in 19s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7668/ : SUCCESS in 3m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1852/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2749/ : SUCCESS in 19m 27s

**Pull request merged**